packages/react-utils/README.md

```diff
@@ -0,0 +1,3 @@
+# @base-ui-components/react-utils
```

Comment:
I think I'd go with just `@base-ui-components/utils` for the name. The package contains more than just react-specific utils, and the name is already long due to the prefix.

Reply:
According to the current [conventions](https://www.notion.so/mui-org/engineering-mui-utils-purpose-9a9fc9da3a004864b6c4e1f4d1f24f95?source=copy_link#d1699bb48f0f4ef58ca831d1ee9772ec), that would make it semver unstable and then it can only be depended in this repo, and only if the dependency is defined with exact version.
I think `@base-ui-components/utils` makes sense, just don't make breaking changes in it on a minor.